vpr/src/place/place_delay_model.cpp

```diff
@@ -170,6 +170,14 @@ float SimpleDelayModel::delay(const t_physical_tile_loc& from_loc, int /*from_pi
         "is disable because VTR_ENABLE_CAPNPROTO=OFF." \
         "Re-compile with CMake option VTR_ENABLE_CAPNPROTO=ON to enable."
 
+void SimpleDelayModel::read(const std::string& /*file*/) {
```

Comment:
Why are these defined and then redefined?

Reply:
The definitions are guarded by a macro that checks whether Cap’n Proto is enabled. If it is not enabled, an error is thrown if these functions are called. Otherwise, Cap’n Proto functions are used to read and write the array.

Reply:
OK, thanks.